app/controllers/admin/cms/collections_controller.rb

```diff
@@ -5,38 +5,38 @@ def show
   end
 
   def new
-    @resource = Collection.new
+    @collection = Collection.new
   end
 
   def create
-    @resource = Collection.new(resource_params)
+    @collection = Collection.new(resource_params)
 
-    if @resource.save
+    if @collection.save
       flash[:notice] = "The new collection has been saved."
       redirect_to admin_cms_path()
     else
       flash[:errors] = "The new collection could not be saved."
-      render :new
+      render :new, status: 422
```

Comment:
you can use the status names here, too, e.g. `render ... status: :not_found` (which does a 404)